jenkins/Jenkinsfile

```diff
@@ -90,7 +137,7 @@ pipeline {
           description: "Is not currently baked into the image, but is here for future functionality")
     string(name: "leonardo_jupyter", defaultValue: "5c51ce6935da",
           description: "The tag of the jupyter image to pull in docker image build. Used in the prepare script.")
-    string(name: "welder_server", defaultValue: "latest",
+    string(name: "welder_server", defaultValue: "6a783a5",
```

Comment:
when will this hash be updated?

Reply:
this is only a default, this PR adds the capability to source the welder hash from reference.conf